ex03.ex

```diff
@@ -76,8 +83,13 @@ defmodule Ex03 do
       true
 
   """
-
-  def list_contains . .. "your code"
+  def list_contains([], _ ), do: false
```

Comment:
## -2

Pattern matching would be a lot tidier here:

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~